src/main/java/com/google/sps/data/Challenge.java

```diff
@@ -0,0 +1,39 @@
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Challenge {
+  private String challenge_type;
```

Comment:
@natalie-abreu @ekengni - Am I correct in understanding that the number of challenge types can only fall within a specific [list](https://github.com/googleinterns/step72-2020/blob/12eb87aeebc33352942ec7b50fe7ca7864f83155/src/main/java/com/google/sps/servlets/EventsServlet.java#L89)?

If so, I suggest we push that list into the data layer (this file), and making challenges_type an Enumeration with only the allowed types. Moreover, change the [src/main/java/com/google/sps/servlets/EventsServlet.java file](https://github.com/googleinterns/step72-2020/blob/12eb87aeebc33352942ec7b50fe7ca7864f83155/src/main/java/com/google/sps/servlets/EventsServlet.java#L158) to use this class to validate the data. In fact, most data validation should be done at the data object level.